tests/components/vacuum/test_device_trigger.py

```diff
@@ -49,21 +51,23 @@ async def test_get_triggers(hass, device_reg, entity_reg):
         config_entry_id=config_entry.entry_id,
         connections={(device_registry.CONNECTION_NETWORK_MAC, "12:34:56:AB:CD:EF")},
     )
-    entity_reg.async_get_or_create(DOMAIN, "test", "5678", device_id=device_entry.id)
+    entity_reg.async_get_or_create(
+        Platform.VACUUM, "test", "5678", device_id=device_entry.id
+    )
     expected_triggers = [
         {
             "platform": "device",
             "domain": DOMAIN,
             "type": "cleaning",
             "device_id": device_entry.id,
-            "entity_id": f"{DOMAIN}.test_5678",
+            "entity_id": f"{Platform.VACUUM}.test_5678",
```

Comment:
I really don't see the need for this all...

Like this is the test of the Vacuum integration, why would it use the Platform enum outside of its package for itself?